src/StringUtility.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2021, Group5 Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#pragma once
+#include <vector>
+#include <string>
+
+namespace StringUtility {
+	std::vector<std::string> split(const std::string& str, const std::string& del);
```

Comment:
Please add function comments, see: https://google.github.io/styleguide/cppguide.html#Function_Comments

Reply:
Also, it is recommended to add usage examples.

Reply:
Also, consider using library for this, such as boost.

Reply:
The implement will use boost::split. Done.